README.md

```diff
@@ -71,7 +71,7 @@ training_cmd="--training_type lora \
   --seed 42 \
   --mixed_precision bf16 \
   --batch_size 1 \
-  --train_steps 2000 \
```

Comment:
So, train shorter with a smaller LR? 👁️

Reply:
Higher learning rate seems to make the model worse somehow when doing stylistic training :/ Yet to find the optimal training configuration for LTXV but ~1000-1500 steps seems to be okay